<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guide</class>
 <widget class="QDialog" name="guide">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>411</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>211</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>가이드 조회</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>104</y>
     <width>211</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>가이드 추가</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>184</y>
     <width>211</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>가이드 수정</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>235</y>
     <width>211</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>가이드 삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>285</y>
     <width>211</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>가이드 상태변경</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>211</width>
     <height>35</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>가이드 이름 입력</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>211</width>
     <height>35</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>가이드 이름 입력</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
